journeys/0xCUBE.md

```diff
@@ -0,0 +1,51 @@
+<p align="center">
+  <img src="images/Small-Cube.png" />
+</p align="center">
+
+<h1 align="center">Hi! I'm 0xCUBE! Welcome to my profile!</h1>
+<p align="center">
+ 
+ <img src="https://badges.pufler.dev/visits/0xCUB3/0xCUB3"/> 
+ <!-- <img src="https://badges.pufler.dev/years/ritik307"/> -->
+ <img src="https://badges.pufler.dev/repos/0xCUB3"/>
+ <img src="https://badges.pufler.dev/commits/yearly/0xCUB3" />
+
+</p>
+<br>
+
+<p align="center">
+  <img src="https://quotes-github-readme.vercel.app/api?type=horizontal&theme=dark" alt="Quote" />
+</p align="center">
+<br>
+
+<h2 align="center">Fun Facts About Me</h2>
+<p>
+  <ul>
+    <li>🧍I am from the US</li>
+    <li>👨‍💻My programming languages are Python, Swift, .sh, and I'm just starting to learn Java.</li>
+    <li>📈I am also learning algorithmic trading with Python!</li>
+    <li>⛷️My favorite sports include swimming, skiing, tennis, and competitive sailing.</li>
+    <li>🤖I occasionally make websites and simple Discord bots (lmk if you want one!)</li>
+  </ul>
+</p>
+<br>
+
+<h2 align="center">My GitHub Stats</h2>
+<p align = "center">
+  <img  src = "https://github-readme-stats.vercel.app/api?username=0xCUB3&show_icons=true&theme=radical">
+  <img src = "https://github-readme-stats.vercel.app/api/top-langs/?username=0xCUB3&theme=radical">
+</p>
+<br>
+
+<h2 align="center">Social Links</h2>
```

Comment:
social links not working


![image](https://user-images.githubusercontent.com/84712013/181920508-c230d5da-f801-4433-9b52-a5c78ce83f12.png)